File: System/Model/src/Model/HeartRate.java
package Model;

public class HeartRate extends Measurement {

    private int beatsPerMin;

    public HeartRate(long patientId, String timestamp, int beatsPerMin) {
        super(patientId, timestamp);
        this.beatsPerMin = beatsPerMin;
    }

    public int getBeatsPerMin() {
        return beatsPerMin;
    }


    public boolean isCritical() {
        return beatsPerMin < 60 || beatsPerMin > 100;
    }

    public String toString() {
        return super.getTimestamp() + "	" + beatsPerMin;
    }
}

File: System/Common/HeartRate.cs
using System;

namespace Common
{
    public class HeartRate : Measurement
    {
        public long PatientID { get; set; }

        public int beatsPerMin { get; set; }

        public HeartRate(long PatientID,string timestamp, int beatsPerMin)
        {
            this.PatientID = PatientID;
            this.Timestamp = timestamp;
            this.beatsPerMin = beatsPerMin;
        }


        public Boolean isCritical()
        {
            if (beatsPerMin < 60 || beatsPerMin > 100)
                return true;
            else
                return false;
        }

    }
}